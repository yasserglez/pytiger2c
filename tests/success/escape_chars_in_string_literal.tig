let
    /* msg1 is: Can you read this? */
    var msg1 := "\067\097\110\032\121\111\117\032\114\101\097\100\032\116\104\105\115\063"
    var msg2 := "\n\"Quoted string\"\^J"
in
    print(msg1);
    print(msg2)
end
