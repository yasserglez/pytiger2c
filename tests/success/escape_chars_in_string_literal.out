Can you read this?
"Quoted string"
